src/fb/ReactTestUtilsFBEntry.js

```diff
@@ -5,14 +5,8 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
- *
- * @providesModule ReactTestUtils
  */
 
 'use strict';
 
-const {
-  __SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED,
-} = require('ReactDOM-fb');
-
-module.exports = __SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.ReactTestUtils;
+module.exports = require('react-dom/test-utils');
```

Comment:
I don't understand this yet. Why does a FB entry point simply re-export? Can't we point the FB entry point to whatever `react-dom/test-utils` truly is?